velox/functions/prestosql/Arithmetic.h

```diff
@@ -487,8 +487,8 @@ struct EulerConstantFunction {
 template <typename T>
 struct TruncateFunction {
   template <typename TInput>
-  FOLLY_ALWAYS_INLINE void call(TInput& result, TInput a) {
-    result = std::trunc(a);
+  FOLLY_ALWAYS_INLINE void call(TInput& result, TInput a, int32_t n = 0) {
```

Comment:
I wonder if we should provide two overloads for these functions for performance, so we can remove the branch from the common case (which is probably a single parameter).

Reply:
@pedroerp Will that imply adding the overload to TruncateFunction class and adding a separate overload registration(unary) in the  registrar?